docs/tutorials/scale-out.md

```diff
@@ -114,28 +116,64 @@ Let's see how it performs given 2 `Replicas`:
 ```
 
 As you can see, now it only takes 3 seconds to finish the task.
-
-Instead of scale an `Executor` at creation time,
-we can scale an `Executor` after `Flow` being created, by calling `scale` method,
-not only scale up, but also scale down.
-For instance,
-you want to scale up and scale down an `Executor`:
-
-```python
-f = Flow().add(name='fast_executor').add(name='slow_executor', uses=MyTokenizer, replicas=2)
-f.scale('slow_executor', replicas=3)  # scale up from 2 to 3
-f.scale('slow_executor', replicas=1)  # scale down from 3 to 1
-```
-
 Quite intuitive, right? In a later section,
 we'll introduce what `replicas` means under different deployment options.
 
 ## Split Data over Machines: Shards
 
 ### Context
 
+Now with your text corpus encoded as TF-IDF embeddings,
+it's time to save the results.
+We'll use Jina's [PQLite](https://hub.jina.ai/executor/pn1qofsj) `Indexer` to persist our embeddings for fast Approximate Nearest Neightbour search.
+
+And you add this `PQLiteIndexer` to your Flow:
+
+```python
+from jina import Flow
+
+f = Flow().add(
+    name='fast_executor').add(
+    name='slow_executor', uses=MyTokenizer).add(
+    name='pqlite_executor', uses='jinahub://PQLiteIndexer', uses_with={
+      'dim': 5215,
+      'metric': 'cosine'
+    },
+    uses_metas={'workspace': 'CHANGE-TO-YOUR-PATH/workspace'},
+    install_requirements=True)
+```
+
 ### Partitioning your Data
 
+Now let's run the `Flow` to index your data:
+```python
+with f:
+    f.post(on='/index', inputs=news_generator, show_progress=True)
+```
+
+The `PQLiteIndexer` will save your indexed `Documents` to your specified `workspace` (directory).
+Since the default number of shards is one.
+All the data will be saved to `YOUR-WORKSPACE-DIR/PQLiteIndexer/0/` and `0` is the shard id.
+
+If you want to distribute your data to different places, Jina allows you to use `shards` to specify number of shards.
```

Comment:
Mention that `Jina` will automaticall `reduce` the `results` to collect `matches` from `different` shards`